Session_2/word_count.js

```diff
@@ -0,0 +1,27 @@
+// Word Count
+// http://www.codewars.com/kata/word-count
+
+
+function countWords(str) {
+//return str.trim().split(/\s+/).length;
+
+// ******** Could you please explain why this is not passing all test cases ************
```

Comment:
This might be using some uni-code chars for spaces in the code to test. Your logic seems fine here, but it is nice to have a solution using regex in such cases. This will have less code and a clean solution